### ISSUE TITLE
The DSN address contains // resulting in ErrorTranspotResult

### ISSUE THREAD

**Issue by gaoffan** [getsentry/sentry-java#1030]
_Platform:_
- [x]  Java -> 1.8
- [x]  Spring -> Spring Boot 2.3

_IDE:_
- [x]  IntelliJ -> 2020.2

_Build system:_
- [x]  Maven

_Platform installed with:_
- [x] Maven Central

The version of the SDK:
**3.1.0**

---
I use self-hosted Sentry and set the ROOT URL to http://www.example.com/ , then the website suggest me to configure Sentry SDK as

```
Sentry.init(options -> {
  options.setDsn("http://xxx@www.example.com//2");
});
```

But I can't receive any error info at Sentry, so I debug sdk and find ErrorTranspotResult(500) in
flush:239, AsyncConnection$EnvelopeSender (io.sentry.transport)

The problem is solved after I deleted the last / in the DSN address.

So can normalize the DSN address before it parsed?

**Comment by maciejwalkowiak** [comment]
Thanks for reporting, fix is in review #1032.

**Comment by bruno-garcia** [comment]
Will go out on the next release.
